assets/html/evolution/drop-nav-carousel.html

```diff
@@ -0,0 +1,503 @@
+<!doctype html>
+<html class="no-js" lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="author" content="oscar rodriguez">
+    <title>Drop Nav Carousel</title>
+    <meta name="description" content="{{DESCRIBE THIS PAGE HERE}}">
```

Comment:
@OscarRobertRodriguez 
Fill out this placeholder: `{{DESCRIBE THIS PAGE HERE}}`